modules/osdk-bundle-deploy-olm.adoc

```diff
@@ -22,9 +25,9 @@ The Operator Bundle Format is the default packaging method for Operator SDK and
 - OpenShift CLI (`oc`) v{product-version}+ installed
 - Operator Lifecycle Manager (OLM) installed on a Kubernetes-based cluster (v1.16.0 or later if you use `apiextensions.k8s.io/v1` CRDs, for example {product-title} {product-version})
 - Logged into the cluster with `oc` using an account with `cluster-admin` permissions
-- Operator built by using the Operator SDK
+- Operator project initialized by using the Operator SDK
 ifdef::golang[]
-- Prepared your Go-based Operator to run on {product-title} by updating the project to use supported images
+- If your Operator is Go-based, your project has been updated to use supported images for running on {product-title}
```

Comment:
I'm not clear on this one. Does the Operator support the images *because* it is Go-based or _If your Operator is Go-based, your project must have been/must be updated to use supported images..._

Reply:
The latter! Fixing.